geo-data-server/src/main/java/com/powsybl/geodata/server/GeoDataService.java

```diff
@@ -60,7 +62,11 @@
         return substationsGeoDataDB;
     }
 
-    public List<SubstationGeoData> getSubstations(Network network, Set<Country> countries) {
+    List<SubstationGeoData> getSubstations(Network network, Set<Country> countries) {
+        String countiesSet = countries.toString();
```

Comment:
I am wondering if it is not a Sonar false alarm as here input is parsed to Country enum and there is no risk of log forging?

Reply:
Same reason as below.

Reply:
Each of the input country is passed to Country.valueOf, if a malicious line break is introduced we will ony have an enum parse exception.

Reply:
Alright,